### PR TITLE
Updating func to 1.7 release

### PR DESCRIPTION
**Pull request by salaboy**
# Changes
- :broom: updating to 1.7 release 

/kind cleanup

Just updating to latest version

**Release Note**

```release-note

```

**Docs**

```docs

```

### CODE REVIEW

File: func.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require 'fileutils'
 
 class Func < Formula
-  v = "v0.25.1"
+  v = "knative-v1.7.0"
```
Comment (lance) [inline func.rb:4]: I think you just need the `v1.7.0` here

Comment (maximilien) [reply func.rb:4]: Yup

**Review by maximilien — COMMENTED** [review]
@salaboy please fix the v field and I'll approve and merge.

File: func.rb [hunk 2]
```diff
@@ -14,14 +14,14 @@ class Func < Formula
...
-      sha256 "84dcbc6245940e5909243bdd93cd35ad2f08737b677a4a043b15f0d278b1c921"
+      sha256 "80d0f6350c5d222259c22531fadddcca86339581efe455149b970648f5a2dd40"
     end
   else
     url "#{base_url}/#{file_name}_linux_amd64"
-    sha256 "e51f45ae6708edf8ad6b980e0045b8fdb49b4d31f3e1a70106ade5d11d20c429"
+    sha256 "1b0f2f9a357cb930a4c256620018b25adb634547a8fb315ddc303f150d4cf7f3"
   end
 
   def install
```
Comment (lance) [inline func.rb:27]: We now also produce `func_linux_arm64` (as well as `_ppc64le` and `_s390x` but I don't know if there is homebrew on those systems???) . Could you add code for this below, similar to what is there in the `if OS.mac?` section but in `OS.linux`?

File: func.rb (hunk 1 above)
Comment (salaboy) [reply func.rb:4]: But, when I looked at the link for downloading the binary is using that 
- https://github.com/knative-sandbox/kn-plugin-func/releases/tag/knative-v1.7.0
- https://github.com/knative-sandbox/kn-plugin-func/releases/download/knative-v1.7.0/func_darwin_amd64

@lance @maximilien

**Comment by salaboy** [comment]
@lance @maximilien added Linux arm64 and checked the links.. please confirm that the version is ok.

File: func.rb (hunk 1 above)
Comment (lance) [reply func.rb:4]: Yes I see that now. It's odd to me that most of the plugins use basic semver, but the `kn` client is the same, using `knative-vX.X.X`. I don't really understand why the difference. 🤷

**Review by lance — COMMENTED** [review]
/lgtm

**Review by maximilien — APPROVED** [review]
/approve
/lgtm

**Pull request merged**